modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account/users.jsp

```diff
@@ -0,0 +1,90 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+SearchContainer accountUsersDisplaySearchContainer = new UserSearch(liferayPortletRequest, renderResponse.createRenderURL());
+
+List<User> results = new ArrayList<>();
+
+accountUsersDisplaySearchContainer.setResults(results);
+accountUsersDisplaySearchContainer.setEmptyResultsMessage("there-are-no-users-associated-to-this-account");
+
+ViewAccountsUsersManagementToolbarDisplayContext viewAccountsUsersManagementToolbarDisplayContext = new ViewAccountsUsersManagementToolbarDisplayContext(request, liferayPortletRequest, liferayPortletResponse, accountUsersDisplaySearchContainer);
+
+String backURL = ParamUtil.getString(request, "backURL", String.valueOf(renderResponse.createRenderURL()));
+
+portletDisplay.setShowBackIcon(true);
+portletDisplay.setURLBack(backURL);
+
+AccountDisplay accountDisplay = (AccountDisplay)request.getAttribute(AccountWebKeys.ACCOUNT_DISPLAY);
+
+renderResponse.setTitle((accountDisplay == null) ? "" : accountDisplay.getName());
+%>
+
+<clay:management-toolbar
+	displayContext="<%= viewAccountsUsersManagementToolbarDisplayContext %>"
+/>
+
+<div class="container-fluid container-fluid-max-xl">
+	<aui:form method="post" name="fm">
+		<aui:input name="userId" type="hidden" />
+
+		<liferay-ui:search-container
+			searchContainer="<%= accountUsersDisplaySearchContainer %>"
+		>
+			<liferay-ui:search-container-row
+				className="com.liferay.portal.kernel.model.User"
+				escapedModel="<%= true %>"
+				keyProperty="userId"
+				modelVar="accountUsers"
+			>
+				<liferay-ui:search-container-column-text
+					cssClass="table-cell-expand-small table-cell-minw-150"
+					name="name"
+					value="<%= accountUsers.getFullName() %>"
+				/>
+
+				<%
+				List<EmailAddress> emailAddresses = accountUsers.getEmailAddresses();
+
+				String emailAddress = null;
+
+				if (!emailAddresses.isEmpty()) {
```

Comment:
Should we us `Validator.isNotNull()` instead of `isEmpty()`?

Reply:
Hi Jon, I went through the backend with Pei and the getEmailAddresses method call will never return a null value but an empty list if no email addresses are found (return Collections.emptyList();)

Reply:
ok sounds good.  thanks for checking.  I know `Validator.isNotNull()` checks for both `null` values and `empty` strings which is why I asked that.